lambdas/process-candidate-identity/src/main/java/uk/gov/di/ipv/core/processcandidateidentity/service/StoreIdentityService.java

```diff
@@ -64,20 +65,27 @@ public void storeIdentity(
 
         LOGGER.info(LogHelper.buildLogMessage("Identity successfully stored"));
 
-        sendIdentityStoredEvent(achievedVot, identityType, auditEventParameters);
+        sendIdentityStoredEvent(strongestMatchedVot, identityType, auditEventParameters);
     }
 
     private void sendIdentityStoredEvent(
-            Vot achievedVot,
+            VotMatchingResult.VotAndProfile strongestMatchedVot,
             CandidateIdentityType identityType,
             SharedAuditEventParameters auditEventParameters) {
+
+        boolean sisRecordCreated =
+                configService.enabled(CoreFeatureFlag.STORED_IDENTITY_SERVICE)
+                        && identityType != CandidateIdentityType.PENDING;
```

Comment:
I don't think we can rely on just the flag - isn't it possible that the flag is enabled but we fail to write the record?